src/c++/perf_analyzer/README.md

```diff
@@ -1,752 +1,167 @@
-- From the File menu select "Import..."
-- Select "Upload" and upload the file
-- Select "Replace data at selected cell" and then select the "Import data" button
-
-### Server-side Prometheus metrics
-
-Perf Analyzer can collect
-[server-side metrics](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md#gpu-metrics)
-, such as GPU utilization and GPU power usage. To enable the collection of these metrics,
-use the `--collect-metrics` CLI option.
-
-Perf Analyzer defaults to access the metrics endpoint at
-`localhost:8002/metrics`. If the metrics are accessible at a different url, use
-the `--metrics-url <url>` CLI option to specify that.
+### Other Features
 
-Perf Analyzer defaults to access the metrics endpoint every 1000 milliseconds.
-To use a different accessing interval, use the `--metrics-interval <interval>`
-CLI option (specify in milliseconds).
+- [Sequence Models](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/architecture.md#stateful-models)
+  and
+  [Ensemble Models](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/architecture.md#ensemble-models)
+  can be profiled in addition to standard/stateless models
 
-Because Perf Analyzer can collect the server-side metrics multiple times per
-run, these metrics are aggregated in specific ways to produce one final number
-per sweep (concurrency/request rate). Here are how they are aggregated:
+- [Input Data](docs/input_data.md) to model inferences can be auto-generated or
+  specified as well as verifying output
 
-| Metric | Aggregation |
-|--------|-------------|
-| GPU Utilization | Averaged from each collection taken during stable passes. We want a number representative of all stable passes. |
-| GPU Power Usage | Averaged from each collection taken during stable passes. We want a number representative of all stable passes. |
-| GPU Used Memory | Maximum from all collections taken during a stable pass. Users are typically curious what the peak memory usage is for determining model/hardware viability. |
-| GPU Total Memory | First from any collection taken during a stable pass. All of the collections should produce the same value for total memory available on the GPU. |
+- [TensorFlow Serving](docs/benchmarking.md#benchmarking-tensorflow-serving) and
+  [TorchServe](docs/benchmarking.md#benchmarking-torchserve) can be used as the
+  inference server in addition to the default Triton server
 
-Note that all metrics are per-GPU in the case of multi-GPU systems.
+<br>
 
-To output these server-side metrics to a CSV file, use the `-f <filename>` and
-`--verbose-csv` CLI options. The output CSV will contain one column per metric.
-The value of each column will be a `key:value` pair (`GPU UUID:metric value`).
-Each `key:value` pair will be delimited by a semicolon (`;`) to indicate metric
-values for each GPU accessible by the server. There is a trailing semicolon. See
-below:
+# Quick Start
```

Comment:
Make sure to link to the docs/quick_start for full details at the end of this section

Reply:
Added 👍